src/Accelerometers/Accelerometers.cpp

```diff
@@ -87,6 +89,15 @@ static void AddLocalAccelerometerRun(unsigned int numDataPoints) noexcept
 	reprap.BoardsUpdated();
 }
 
+static void AddLocalAccelerometerAverages(float averages[]) noexcept
```

Comment:
Why not add extra arguments to AddLocalAccelerometerRun instead of adding this function?

Reply:
I will do the same thing for `AddAccelerometerRun` as well